sdk/search/search-documents/test/public/node/searchClient.spec.ts

```diff
@@ -17,40 +17,37 @@ import {
   AzureKeyCredential,
 } from "../../../src";
 import { Hotel } from "../utils/interfaces";
-import { createIndex, populateIndex, WAIT_TIME, createRandomIndexName } from "../utils/setup";
+import { createIndex, populateIndex, WAIT_TIME } from "../utils/setup";
 import { delay, serviceVersions } from "../../../src/serviceUtils";
 import { versionsToTest } from "@azure/test-utils";
 
-const TEST_INDEX_NAME = isLiveMode() ? createRandomIndexName() : "hotel-live-test1";
```

Comment:
Could you please expand why this is removed? The CreateRandonIndexName method should be called only for live mode. But now it seems it is getting called even for the playback mode which will cause issues

Reply:
< Discussed offline 😎 >